home/screens/DiagnosticsScreen.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import { StyleSheet, View } from 'react-native';
+
+import BouncingShadowButton from '../components/BouncingShadowButton';
+import ScrollView from '../components/NavigationScrollView';
+import { StyledText } from '../components/Text';
+import Colors from '../constants/Colors';
+import Environment from '../utils/Environment';
+
+export default function DiagnosticsScreen({ navigation }) {
+  return (
+    <View style={{ flex: 1, backgroundColor: Colors.light.greyBackground }}>
```

Comment:
theme?

Reply:
This view is unused, the background color is defined in the scroll view. Deleted the wrapper view